Milestone emails templates and sending implementation

aileen opened the pull request:
refs https://www.notion.so/ghost/Marketing-Milestone-email-campaigns-1d2c9dee3cfa4029863edb16092ad5c4?pvs=4

- Added email template for milestones with using a configuration file for different member milestone values, as we're sending different content for each one
- Implement sending the email to users who have `milestone-notifications` enabled, currently still behind a flag

---
### Todos before merging:
- [x] Finish email template design
- [x] Ship with public beta

aileen committed:
Added Milestone email sending to staff service

refs https://www.notion.so/ghost/Marketing-Milestone-email-campaigns-1d2c9dee3cfa4029863edb16092ad5c4?pvs=4

- Hooks up `MilestoneCreatedEvent` to send emails when intended to
    - User notification setting is set to true
    - The event states that an email should be send
- Added email templates for ARR and Member emails utilised by a config with different contect for each Member milestone email
- Added and updated tests

aileen committed:
Moved milestone emails to public beta

JBTAdmin commented on ghost/staff-service/lib/emails.js:
```diff
@@ -170,13 +170,50 @@ class StaffServiceEmails {
      * @returns {Promise<void>}
      */
     async notifyMilestoneReceived({milestone}) {
+        if (!milestone?.emailSentAt || (milestone?.meta && milestone?.meta?.reason)) {
```
Do we really need `milestone?.meta` before && operator?

JBTAdmin commented on ghost/staff-service/lib/emails.js:
```diff
@@ -170,13 +170,50 @@ class StaffServiceEmails {
+
+        const emailPromises = [];
         const users = await this.models.User.getEmailAlertUsers('milestone-received');
+        const formattedValue = this.getFormattedAmount({currency: milestone?.currency, amount: milestone.value, maximumFractionDigits: 0});
```
Don't know the whole context but for `currency: milestone?.currency` you are using optional chaining on the other hand for `amount: milestone.value` you are not. Which looks odd to me.

aileen committed:
Removed unneeded check

aileen committed:
Updated milestone email template with updated design

aileen committed:
Added milestone image urls

- Updated the milestone email config to return correct image for ARR miilestones
- Added an early exit when no configuration is found for a milestone value, so we don't send out half empty emails. Send a warning log in those cases.
- Added all correct image urls
- Added option to handle passed in `height` property for images

aileen committed:
Make it easier to test milestones in development mode

aileen committed:
Bold accents on number of members in mail content

peterzimon committed:
Added animated logo in email header

aileen committed:
Updated ARR titles for large numbers to not wrap lines

aileen committed:
Updated email copy

SimonBackx reviewed (approved):
One tiny comment, but apart from that everything looks good! 👌

aileen committed:
Removed orphaned quote
